biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/FeatureInterface.java

```diff
@@ -169,25 +169,35 @@
      */
     public void setUserObject(Object userObject);
 
-    
-	/**
-     * Get the qualifiers for this feature
-     * @return
-     */
-
-    public Map<String, List<Qualifier>> getQualifiers();
-
-    /**
-     * Set the qualifiers
-     * @param qualifiers
-     */
-
-    public void setQualifiers(Map<String, List<Qualifier>> qualifiers);
-    /**
-     * Add a qualifier
-     * @param qualifier
-     */
-
-    public void addQualifier(String key, Qualifier qualifier);
-
+    /* new feature interface methods for accessing qualifers and the new map 
+     * see abstract feature for implementation 
+     */
+    public GenBankQualifierMap getQualifierMap();
+    public Qualifier[] getQualifiers();
+	public void setQualifierMap(GenBankQualifierMap qualifierMap);
+	public void setQualifiers(Qualifier[] qualifiers);
+	public void addQualifier(Qualifier qualifier);
+	public void addQualifiers(Qualifier[] qa);
+    public Qualifier getQualifierByName(String qName);
+    public Qualifier getFirstQualifierByValue(String value);
+    public Qualifier[] getQualifiersByValue(String value);
+	//DBreferenceInfo 
+	public String[][] getAllDatabasesReferenceInfos();
+	public String[] getAllDatabases();
+	public String[] getAllDatabaseReferences();
+	public String[] getDatabaseReferenceInfo(int i);
+	public String getDatabase(int i);
+	public String getDatabaseReference(int i);
+	public String[] getFirstDatabaseReferenceInfo();
+	public String getFirstDatabaseReference();
+	public String getFirstDatabase();
+	public String getDatabaseReference(String database, int i);
+	public String[] getAllDatabaseReferences(String database);
+	public String getFirstDatabaseReference(String database);
+	public void setDatabaseReferenceInfo(String database, String reference);
+	public boolean addDatabaseReferenceInfos(String[][] entries);
+	//old stuff to be removed
+	@Deprecated //use addQualifier(Qualifier q)
```

Comment:
Can you add some javadoc for the deprecated method to point to which one to use instead